bridge/jni/src/api.rs

```diff
@@ -0,0 +1,29 @@
+use jni::{
+	objects::{JClass, JObject, JString},
+	JNIEnv,
+};
+
+#[no_mangle]
+pub extern "system" fn Java_frequency_Native_hello<'local>(
```

Comment:
We probably don't need `extern "system"` here--that's more for interacting with OS libraries and system calls. For exposing Rust functions to other languages via a C-compatible ABI, just `extern "C"` is the way.

Reply:
Thius is the way javac -h generates the jni functions and even if you look at their example you see the same https://github.com/jni-rs/jni-rs/blob/master/example/mylib/src/lib.rs#L33. I think both of them work but I'm not sure which way is better.

Reply:
I think that will break on Win32. Everything I've looked up says JNI expects a "C" ABI. "system" maps to "stdcall" on Win32. We would only want that if calling WIn32 system libraries from Rust code.

On any platform *but* Win32, `extern "C"` == `extern "system"`

Reply:
I think you would only use "system/stdcall" if you were:
* declaring an external Win32-native DLL function for Rust to be able to call
* writing a native Win32 Rust library that other native Win32 DLLs or apps (not JNI) will call using the platform native ABI